frontend/server/src/Authorization.php

```diff
@@ -217,6 +217,23 @@ public static function canEditClarification(
                 )));
     }
 
+    public static function isUnderThirteenUser(\OmegaUp\DAO\VO\Users $user): bool {
+        // This is mostly for users who hasn't give us their birth day
+        if (is_null($user->birth_date)) {
+            return false;
+        }
+        // User's age is U13? $user->birth_date - current date then return true, otherwise return false
+        if (
+            $user->birth_date <= strtotime(
+                '-13 year',
+                \OmegaUp\Time::get()
+            )
+        ) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return $user->birth_date <= strtotime(
            '-13 year',
            \OmegaUp\Time::get()
        );
```

no need for the `if`/`else`!

Reply:
okk